docs/UserGuide.md

```diff
@@ -1446,12 +1662,61 @@ To get a list of `COMMAND_WORD`, do `help` with no arguments
 > Example: help add_transaction
 > ```
 
-#### Reset UniCa$h
+#### Reset UniCash
+
 Resets UniCa$h to its default state.
 
 Command: `reset_unicash`
 
-Command Words Accepted: `reset_unicash` (case-insensitive)
+Command Words Accepted: `reset_unicash` only (case-insensitive)
+
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+Similar to `Clear Transactions`, this command overwrites all existing transactions in UniCa$h.
+Thus, as an added layer of safety, the command has no short form alternatives.
+It also cannot be followed with any arguments, options, or non-whitespace text.
+</div>
+
+##### Successful Execution
+
+###### Example 1
+
+> **Case**: Command entered with correct format
+>
+> **Input**: `reset_unicash`
+>
+> **Output**:
+> ```
+> UniCa$h has been successfully restored to its original state!
+> ```
+>
+> Note: The `Transactions List` in the UI output for this command will look
+> identical to the very first image of UniCa$h shown at the top of the user
+> guide, as it is meant to restore UniCa$h to its default state.
+
+###### Example 2
+
+> **Case**: Command entered with acceptable alternate format
+>
+> **Input**: `RESET_UNICASH`
+>
+> **Output**:
+> ```
+> UniCa$h has been successfully restored to its original state!
+> ```
```

Comment:
Same comment as clear transactions where this example is already covered with our pre-empt that all command words are case insensitive

Reply:
Fixed